### PR TITLE
Feature/special events

### PR DESCRIPTION
**Pull request by dmiddlecamp**
- Needs proofing / style / layout checks

### CODE REVIEW

File: src/content/reference/api.md [hunk 1]
```diff
@@ -177,6 +177,142 @@ we'll give you lots of notice and a clear upgrade path.
...
+```
+    
+    
+#### Safe-Mode
+
+If your device is running an app that needs a particular version of system firmware, your device may come online and
+report that it's in Safe Mode.  In this case, your device is waiting to run your app until it has received an update.
+Some products can receive system updates automatically while in safe mode, but others like the Electron prevent this to
+save you costs on bandwidth.  If you do get an automatic update, you may see 
+an "spark/safe-mode-updater/updating" event.
```
Comment (kennethlimcp) [inline src/content/reference/api.md]: "see a"

**Comment by jme783** [comment]
@dmiddlecamp is this ready to be merged?

**Comment by dmiddlecamp** [comment]
The content is all there, but I was hoping for a second set of eyes to make sure formatting / phrasing / voice was okay

**Comment by jenesaisdiq** [comment]
Looks fine to me, made a quickie fix, going to merge as soon as travis passes @dmiddlecamp

**Pull request merged**